File: xsolla-login-sdk/src/main/java/com/xsolla/android/login/entity/request/AuthUserBody.java
package com.xsolla.android.login.entity.request;

public class AuthUserBody {
    private String username;
    private String password;
    private boolean remember_me;

    public AuthUserBody(String username, String password) {
        this(username, password, false);
    }

    public AuthUserBody(String username, String password, boolean remember_me) {
        this.username = username;
        this.password = password;
        this.remember_me = remember_me;
    }

}

File: xsolla-login-sdk/src/main/java/com/xsolla/android/login/entity/request/AuthUserBody.kt
package com.xsolla.android.login.entity.request

import com.google.gson.annotations.SerializedName

internal data class AuthUserBody(
    val username: String,
    val password: String,
    @SerializedName("remember_me")
    val rememberMe: Boolean
) {
    constructor(username: String, password: String): this(username, password, false)
}